packages/turf-inside/README.md

```diff
@@ -13,53 +13,18 @@ be convex or concave. The function accounts for holes.
 **Examples**
 
 ```javascript
-var pt1 = {
-  "type": "Feature",
-  "properties": {
-    "marker-color": "#f00"
-  },
-  "geometry": {
-    "type": "Point",
-    "coordinates": [-111.467285, 40.75766]
-  }
-};
-var pt2 = {
-  "type": "Feature",
-  "properties": {
-    "marker-color": "#0f0"
-  },
-  "geometry": {
-    "type": "Point",
-    "coordinates": [-111.873779, 40.647303]
-  }
-};
-var poly = {
-  "type": "Feature",
-  "properties": {},
-  "geometry": {
-    "type": "Polygon",
-    "coordinates": [[
-      [-112.074279, 40.52215],
-      [-112.074279, 40.853293],
-      [-111.610107, 40.853293],
-      [-111.610107, 40.52215],
-      [-112.074279, 40.52215]
-    ]]
-  }
-};
-
-var features = {
-  "type": "FeatureCollection",
-  "features": [pt1, pt2, poly]
-};
-
-//=features
-
-var isInside1 = turf.inside(pt1, poly);
-//=isInside1
-
-var isInside2 = turf.inside(pt2, poly);
-//=isInside2
+var pt = point([-77, 44]);
```

Comment:
Fine. I remove commit on "updating readmes" (put in in another PR to be discussed separately)? I keep the 2 other commits in this PR?